src/models/CommonSchema.ts

```diff
@@ -1,6 +1,6 @@
 import * as Joi from "joi";
 
-export const defectsCommonSchema = {
+export const defectsCommonSchema = Joi.object().keys({
```

Comment:
If you look at Joi's docs, this is the correct way to extend a schema and it gives you access to the schema API later if you need it. I'm actually re-using the existing validation schemas when updating a vehicle and I need to do some very small adjustments after I import the schema (see testResultsService line 131-132). This wasn't possible with the old way of extending the schemas, because you didn't have access to the schema API due to the fact that it wasn't a Joi object. It doesn't have any impact on the existing functionality, it's just the correct way to do it.